src/utils/global-D3-tests.js

```diff
@@ -6,7 +6,7 @@ function isToolTipHidden(tooltip) {
     // display of none, a type of hidden, or height/width set to 0.
     // If the element is hidden with opacity=0 or visibility=hidden, jQuery's
     // :hidden will return false because it takes up space in the DOM.
-    // This test combines jQuery's :hidden with tests for opacity and visbility
+    // This test combines jQuery's :hidden with tests for opacity and visibility
```

Comment:
I can remove this from this PR, but I won't do new PR for this one.

Reply:
I will leave it in this one. But it's the 2nd time I've mentioned and it's an important discipline to keep PRs clean and separate. It takes extra work for sure. But not doing that work creates problems when something unrelated was fixed, and then we have to reverse a PR. Because then the stuff that was fixed also gets reversed. It takes less than 1 minute to create a new branch and make the fixes there so I always just do it.